lib/utils/colors.dart

```diff
@@ -3,8 +3,27 @@ import 'dart:ui';
 class TBColor {
   TBColor._();
 
-  static const lightBlue = Color(0xFF34A4E3);
-  static const lightBlueGrey = Color(0xFFC9D2DE);
-  static const grey = Color(0xFF7B828A);
-  static const buttonBorderColor = Color(0xFFCFCFCF);
+  static _TBAppColor app = _TBAppColor();
+  static _TBButtonColor button = _TBButtonColor();
+  static _TBRatingColor rating = _TBRatingColor();
+}
+
+class _TBAppColor {
+  final lightBlue = const Color(0xFF34A4E3);
+  final lightBlueGrey = const Color(0xFFC9D2DE);
+  final grey = const Color(0xFF7B828A);
+}
+
+class _TBButtonColor {
+  final borderColor = const Color(0xFFCFCFCF);
+}
+
+class _TBRatingColor {
+  final yellow = const Color(0xFFFFC046);
+  final grey = const Color(0xFFC0C0C0);
+  final gradientColors = [
+    const Color(0xFFFFC046),
+    const Color(0xFFFFC046),
```

Comment:
Why do you need 2 identical colors

Reply:
It there is only one, then it will gradually go from yellow to grey. This way we will have a sudden change and a pronounced border between the yellow and the grey.

Reply:
Ok, my bad.